opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -258,23 +262,21 @@ message MetricDescriptor {
     Summary summary = 7;
   }
 
-  // Temporality is the temporal quality values of a metric have. It
-  // describes how those values relate to the time interval over which they
-  // are reported.
-  enum Temporality {
-    // INVALID_TEMPORALITY is the default Temporality, it MUST not be
+  // ReportStrategy defines how a metric reports aggregated values. It describes
+  // how those values relate to the time interval over which they are reported.
+  //
+  // Because different backend systems may or may not support different
+  // reporting strategies, it is important that this is used only as an
+  // optimization (avoid keeping the cumulative state on the producer), and only
+  // for known aggregations that can support transformation between the
+  // different strategies.
+  enum ReportStrategy {
```

Comment:
Another approach is to make this "AggregationInterval" - Delta, Arbitrary (used to report aggregation of last 10mins of data), Cumulative and make it more generic to support not only delta and cumulative.

Reply:
This sounds reasonable to me.

Reply:
Just to clarify, "Arbitrary" was just an idea that we can add in the future not now. It is just to show that there is a possibility to extend the options in this enum.

An alternative I heard was to keep `ReportStrategy` and if needed add later a `MOVING_WINDOW` strategy.

Reply:
yeah, not sure about the name, but the *idea* of having an arbitrary window option seems like a good idea.

Reply:
`AggregationInterval` sounds like a message with the timestamps in it to me. Maybe `AggregationStrategy`?

Reply:
Done, using AggregationStrategy.